bindings/python/iota_sdk/types/issuance.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2024 IOTA Stiftung
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import List, Optional
+from dataclasses import dataclass, field
+from dataclasses_json import config
+from iota_sdk.types.common import HexStr, json, SlotIndex
+from iota_sdk.types.slot import SlotCommitment
+
+
+@json
+@dataclass
+class IssuanceBlockHeader:
+    """Information that is used to attach a block in the network.
+    Response of GET /api/core/v3/blocks/issuance
+
+    Attributes:
+        strong_parents: Blocks that are strongly directly approved.
+        weak_parents: Blocks that are weakly directly approved.
+        shallow_like_parents: Blocks that are directly referenced to adjust opinion.
+        latest_parent_block_issuing_time: Latest issuing time of the returned parents.
+        latest_finalized_slot: The slot index of the latest finalized slot.
+        latest_commitment: The latest slot commitment.
```

Comment:
Would match the order of the fields in this class
```suggestion
        strong_parents: Blocks that are strongly directly approved.
        latest_parent_block_issuing_time: Latest issuing time of the returned parents.
        latest_finalized_slot: The slot index of the latest finalized slot.
        latest_commitment: The latest slot commitment.
        weak_parents: Blocks that are weakly directly approved.
        shallow_like_parents: Blocks that are directly referenced to adjust opinion.
```